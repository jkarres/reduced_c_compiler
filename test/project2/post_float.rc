float gi = 0;

float gj = gi++;

float gk = gi++;

function : void main() {
  cout << 1 << " " << gi << endl;
  cout << 2 << " " << gj << endl;
  cout << 3 << " " << gk << endl;

  float li = 0;
  float lj = 10 + li++;
  float lk = li++ + 10;

  cout << 4 << " " << li << endl;
  cout << 5 << " " << lj << endl;
  cout << 6 << " " << lk << endl;

}
