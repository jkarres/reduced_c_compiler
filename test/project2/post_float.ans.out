1 2.00
2 0.00
3 1.00
4 2.00
5 10.00
6 11.00
